cve_bin_tool/checkers/libxslt.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for libxslt
+https://www.cvedetails.com/product/14676/Xmlsoft-Libxslt.html?vendor_id=1962
+"""
+from . import Checker
+
+
+class LibxsltChecker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = [r"libxlst.so."]
+    VERSION_PATTERNS = [r"libxslt ([0-9]+\.[0-9]+\.[0-9]+)"]
+    VENDOR_PRODUCT = [("Xmlsoft", "Libxslt")]
```

Comment:
The matching is case sensitive so you'll want to make those lowercase.
```suggestion
    VENDOR_PRODUCT = [("xmlsoft", "libxslt")]
```